### DISCUSSION TITLE
A galley of users screenshots

### DISCUSSION THREAD

**Discussion by lichingyun** [tanujnotes/Olauncher#165] [Ideas]
Hi,
I have an idea... what about to create a gallery of screenshots of the homescreens from the users?
I don't have a twitter/ facebook/ instagram account so I cannot view/ login the contents from the olauncher pages in these sites.
If is possible the gallery can be hosted in the Tanuj Buymeacoffe pages.
What do you think about?

**Comment by tanujnotes** [comment]
Great idea. Olauncher does have an Instagram page for this: www.instagram.com/olauncherapp
I'll also add a gallery on Olauncher website whenever it goes live.

**Reply by lichingyun** [reply]
Thanks Tanuj can't wait 🤩
